apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/CsvImporterImpl.java

```diff
@@ -242,10 +242,10 @@ private long importCsv(String tableName, int batchLimit, boolean cleanTarget,
                 }
                 rsCounter++;
             }
-            dataSource.commit(); // final commit
+            dataSource.commit(false); // final commit
```

Comment:
Maybe add an option to begin transaction, if not started yet?

Reply:
There is not a problem with starting new transaction, problem with not closing current connection, because external code depends on it and expects it's opened after importer.
I changed that logic because I tried different ways and decided that as most simple one.